modules/cert-manager-enable-metrics.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-monitoring.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-enable-metrics_{context}"]
+= Enabling monitoring for cert-manager
```

Comment:
To avoid repetition of this title on the same page,
let's make this sub-title as "Setting up ServiceMonitor for cert-manager"